mirdata/datasets/beatles.py

```diff
@@ -83,38 +86,32 @@ def __init__(self, track_id, data_home):
 
         self._data_home = data_home
         self._track_paths = DATA.index["tracks"][track_id]
-        self.beats_path = core.none_path_join(
-            [self._data_home, self._track_paths["beat"][0]]
-        )
+        self.beats_path = core.none_path_join([self._data_home, self._track_paths["beat"][0]])
         self.chords_path = os.path.join(self._data_home, self._track_paths["chords"][0])
-        self.keys_path = core.none_path_join(
-            [self._data_home, self._track_paths["keys"][0]]
-        )
-        self.sections_path = os.path.join(
-            self._data_home, self._track_paths["sections"][0]
-        )
+        self.keys_path = core.none_path_join([self._data_home, self._track_paths["keys"][0]])
+        self.sections_path = os.path.join(self._data_home, self._track_paths["sections"][0])
         self.audio_path = os.path.join(self._data_home, self._track_paths["audio"][0])
 
         self.title = os.path.basename(self._track_paths["sections"][0]).split(".")[0]
 
     @core.cached_property
-    def beats(self):
+    def beats(self) -> Optional[annotations.BeatData]:
```

Comment:
quick note about napoleon - we might be able to remove the types from the Cached Properties in the docstring above but we'd have to check